src/Processing.js

```diff
@@ -7997,18 +7997,20 @@
      * <b>ellipseMode()</b> function.
      * The <b>start</b> and <b>stop</b> parameters specify the angles
      * at which to draw the arc.
-     *
+     * There is an option to draw an arc; the rendering technique used is defined by the optional seventh paramter.
+     * Currently we only support CHORD.
      * @param {float} a       x-coordinate of the arc's ellipse
      * @param {float} b       y-coordinate of the arc's ellipse
      * @param {float} c       width of the arc's ellipse
      * @param {float} d       height of the arc's ellipse
      * @param {float} start   angle to start the arc, specified in radians
      * @param {float} stop    angle to stop the arc, specified in radians
+     * @param {enum}  mode    drawing mode, currently only CHORD is available.
      *
      * @see #ellipseMode()
      * @see #ellipse()
      */
-    p.arc = function(x, y, width, height, start, stop) {
+    p.arc = function(x, y, width, height, start, stop, mode) {
       if (width <= 0 || stop < start) { return; }
```

Comment:
default behaviour in Processing is that if there is no mode specified, the ellipse renders as if mode was CHORD, so we'll need a `mode = mode || PConstants.CHORD;` here to ensure it's the right value if not specified by the user